landing_page.css

```diff
@@ -1,4 +1,8 @@
 html {background-color:#264653}
 h1{
     color: #F4A261;
+}
+
+h2{
```

Comment:
can we used a different color?

Reply:
No, I think this looks better for the website

Reply:
okie